x-pack/plugins/observability/public/hooks/slo/use_fetch_slo_list.ts

```diff
@@ -56,56 +47,53 @@ export function useFetchSloList({
   const queryClient = useQueryClient();
   const [stateRefetchInterval, setStateRefetchInterval] = useState<number>(SHORT_REFETCH_INTERVAL);
 
-  const { isInitialLoading, isLoading, isError, isSuccess, isRefetching, data, refetch } = useQuery(
-    {
-      queryKey: sloKeys.list({ kqlQuery, page, sortBy, sortDirection }),
-      queryFn: async ({ signal }) => {
-        const response = await http.get<FindSLOResponse>(`/api/observability/slos`, {
-          query: {
-            ...(kqlQuery && { kqlQuery }),
-            ...(sortBy && { sortBy }),
-            ...(sortDirection && { sortDirection }),
-            ...(page && { page }),
-          },
-          signal,
-        });
+  const { isInitialLoading, isLoading, isError, isSuccess, isRefetching, data } = useQuery({
+    queryKey: sloKeys.list({ kqlQuery, page, sortBy, sortDirection }),
+    queryFn: async ({ signal }) => {
+      const response = await http.get<FindSLOResponse>(`/api/observability/slos`, {
+        query: {
+          ...(kqlQuery && { kqlQuery }),
+          ...(sortBy && { sortBy }),
+          ...(sortDirection && { sortDirection }),
+          ...(page && { page }),
+        },
+        signal,
+      });
 
-        return response;
-      },
-      keepPreviousData: true,
-      refetchOnWindowFocus: false,
-      refetchInterval: shouldRefetch ? stateRefetchInterval : undefined,
-      staleTime: 1000,
-      retry: (failureCount, error) => {
-        if (String(error) === 'Error: Forbidden') {
-          return false;
-        }
-        return failureCount < 4;
-      },
-      onSuccess: ({ results }: FindSLOResponse) => {
-        queryClient.invalidateQueries({ queryKey: sloKeys.historicalSummaries(), exact: false });
-        queryClient.invalidateQueries({ queryKey: sloKeys.activeAlerts(), exact: false });
-        queryClient.invalidateQueries({ queryKey: sloKeys.rules(), exact: false });
+      return response;
+    },
+    cacheTime: 0,
```

Comment:
💬  Only change is here.
Added cacheTime, removed staleTime and keepPreviousData